server/methods/core/static-pages.js

```diff
@@ -0,0 +1,45 @@
+import { Meteor } from "meteor/meteor";
+import { check } from "meteor/check";
+import { StaticPages } from  "/lib/collections";
+import * as Schemas from "/lib/collections/schemas";
+
+Meteor.methods({
+  "insertPage"(title, slug, content, shopId, pageOwner) {
```

Comment:
Thanks for the observation but the syntax I used is a valid ES6 syntax and I don't think it is part of RC style guide.